pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -60,6 +60,7 @@ type changeAggregator struct {
 	kvFeedMemMon *mon.BytesMonitor
 
 	// encoder is the Encoder to use for key and value serialization.
+	// Maybe nil if using 'native' format.
```

Comment:
Will it be `nil`? Won't it be the dummy `nativeEncoder` (which should not be used, but still not nil?)